x-pack/plugins/security_solution/public/timelines/components/field_renderers/field_renderers.tsx

```diff
@@ -260,25 +260,31 @@ MoreContainer.displayName = 'MoreContainer';
 export const DefaultFieldRendererOverflow = React.memo<DefaultFieldRendererOverflowProps>(
   ({ idPrefix, moreMaxHeight, overflowIndexStart = 5, render, rowItems }) => {
     const [isOpen, setIsOpen] = useState(false);
+    const handleClose = useCallback(() => setIsOpen(!isOpen), [isOpen]);
```

Comment:
To avoid new reference of handleClose
```suggestion
    const handleClose = useCallback(() => setIsOpen((prevIsOpen) => !prevIsOpen), []);
```